workloads/kube-burner/common.sh

```diff
@@ -12,7 +12,7 @@ if [[ ${INDEXING} == "false" ]]; then
   unset PROM_URL
 else
   if [[ ${HYPERSHIFT} == "false" ]]; then
-    export PROM_TOKEN=$(oc sa get-token -n openshift-monitoring prometheus-k8s || oc sa new-token -n openshift-monitoring prometheus-k8s)
+    export PROM_TOKEN=$(oc sa get-token -n openshift-monitoring prometheus-k8s || oc create token -n openshift-monitoring prometheus-k8s)
```

Comment:
Hey @paigerube14 , can you swap the order of the commands to prevent printing the warning message?

```suggestion
    export PROM_TOKEN=$(oc create token -n openshift-monitoring prometheus-k8s || oc sa get-token -n openshift-monitoring prometheus-k8s)
```

Reply:
Updated! Thanks